presto-open-telemetry/src/main/java/com/facebook/presto/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -155,6 +156,32 @@ public void startBlock(String blockName, String annotation)
         }
     }
 
+    /**
+     * Create new span with Open Telemetry tracer
```

Comment:
```suggestion
Creates a new span
```
we already are in the OpenTelemetryTracer class

Also add a 
```
 * @throws PrestoException if a blockName already is already used in the existing span map
```

Reply:
Can we keep the javadocs only on the SPI `Tracer` class ?